lib/document.proto

```diff
@@ -2,6 +2,45 @@ syntax="proto3";
 
 package hwsc;
 
+message Publisher {
+    // @inject_tag: bson:"lastName"
+    string last_name = 1;
+    // @inject_tag: bson:"firstName"
+    string first_name = 2;
+}
+
+message Region {
+    // @inject_tag: bson:"city"
+    string city = 1;
+    // @inject_tag: bson:"country"
+    string country = 2;
+}
+
+// Response for ListDistinctFieldValues
+// Request for QueryDocument
+message QueryTransaction {
+    // @inject_tag: bson:"publishers"
+    repeated Publisher publishers = 1;
+    // @inject_tag: bson:"regions"
+    repeated Region regions = 2;
+    // @inject_tag: bson:"callTypes"
+    repeated string call_types = 3;
+    // @inject_tag: bson:"groundTypes"
+    repeated string ground_types = 4;
+    // @inject_tag: bson:"sensorTypes"
+    repeated string sensor_types = 5;
+    // @inject_tag: bson:"sensorNames"
+    repeated string sensor_names = 6;
+    // @inject_tag: bson:"minSampleRate"
+    uint32 min_sample_rate = 7;
```

Comment:
we don't need lines 34 - 37 anymore (min/max sample rate)
Kerri said we don't need users querying sample rates